### DISCUSSION TITLE
Pemilihan State Management Di ReactJS

### DISCUSSION THREAD

**Discussion by ikhlasdansantai** [deaafrizal/forum#29] [Troubleshooting Room] [answered]
Saya sekarang sedang garap **proyek travel website** sederhana pake **nextJS**, cuman saya bingung untuk pemilihan state management nya pake apa, katanya klo memang proyek nya sederhana, pake **Zustand** aja, dan kebetulan sayapun sering pake **Zustand**, cuman ketika scale proyek nya semakin besar, apakah saya harus switch ke **Redux** atau pake **Zustand** aja udah cukup, sekian hatur nuhun.

**Comment by deaafrizal** [comment, marked as answer]
kalau dari awal tau projeknya akan scale jauh, pertimbangan gue lebih ke redux IMHO. 
kecuali gue dari awal dah tau projeknya fix feature, gue pake zustand.
